packages/apriconv.rb

```diff
@@ -3,9 +3,9 @@
 class Apriconv < Package
   description 'a portable implementation of the iconv() library'
   homepage 'http://apr.apache.org/'
-  version '1.2.1-1'
-  source_url 'http://apache.claz.org/apr/apr-iconv-1.2.1.tar.bz2'
-  source_sha256 'c46c919bc2a36a705f91f4dea444b18a83236eef97a417528a988113b3a7e46e'
+  version '1.2.2'
+  source_url 'http://apache.claz.org/apr/apr-iconv-1.2.2.tar.bz2'
```

Comment:
This is available via https. Please switch the protocol.

Reply:
Agreed.  Will fix.